imap_processing/codice/constants.py

```diff
@@ -796,4 +793,5 @@
     252: 6815744,
     253: 7340032,
     254: 7864320,
+    255: 999999,
```

Comment:
One more 9 so it is always increasing? Is there a way to make these a fill_value instead if that is needed?

Reply:
Good catch, i did intend for this to be `9999999`. I need to ask Joey about what to do here, so this is a temporary solution this this problem. I added a TODO here to make sure I don't forget.